packages/firestore/src/local/local_serializer.ts

```diff
@@ -132,11 +132,18 @@ export class LocalSerializer {
     } else {
       queryProto = this.remoteSerializer.toQueryTarget(queryData.query);
     }
-    assert(
-      typeof queryData.resumeToken === 'string',
-      'Persisting non-string resume token not supported.'
-    );
-    const resumeToken = queryData.resumeToken as string;
+
+    let resumeToken: string;
+
+    if (queryData.resumeToken instanceof Uint8Array) {
+      assert(
+        process.env.USE_MOCK_PERSISTENCE === 'YES',
+        'Persisting non-string stream tokens is only supported with mock persistence .'
+      );
+      resumeToken = queryData.resumeToken.toString();
+    } else {
+      resumeToken = queryData.resumeToken;
+    }
```

Comment:
Similar feedback as elsewhere.  If turning resume tokens into strings works, perhaps we should just do that as soon as we get them...  Add a TODO or log a bug?

Reply:
Added TODO pointing to the bug.